man/add_colby_multivar.Rd

```diff
@@ -34,7 +34,7 @@ library(dplyr)
 ANL <- DM \%>\% mutate(value = rnorm(n()), pctdiff = runif(n()))
 
 # TODO: fix
- build_table(l, ANL)
+# build_table(l, ANL)
```

Comment:
As above, I believe this is fixed so please rebase (or whatever the term is) the PR.